updates/1/updates.sql

```diff
@@ -1 +1,24 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL);
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 11155251, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", 18, NULL);
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", "C", "Alchemist", "MIST", 11804891, NULL, "alchemist", "mist", "0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB");
+UPDATE assets SET swapped_for="_ceth_0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0" WHERE identifier="_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5";
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0", "C", "Tellor", "TRB", 11895800, NULL, "tellor", "trb", "0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0");
+UPDATE assets SET swapped_for="_ceth_0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85" WHERE identifier="_ceth_0x1D287CC25dAD7cCaF76a26bc660c5F7C8E2a05BD";
```

Comment:
Perhaps it would make sense to start adding comments explaining what each addition does: https://sqlite.org/lang_comment.html

i don't want to have to check each etherscan address manually

Reply:
Do you mean for each token/asset add a comment saying `Addition of token 0x...`?

Reply:
Could make sense to have something like `/* Introduce TRB token swap*/